src/backend/distributed/sql/citus--8.0-1.sql

```diff
@@ -924,7 +924,7 @@ $function$;
 CREATE FUNCTION master_dist_local_group_cache_invalidate()
     RETURNS trigger
     LANGUAGE C
-    AS 'MODULE_PATHNAME', $$master_dist_local_group_cache_invalidate$$;
+    AS 'MODULE_PATHNAME', $$citus_dist_local_group_cache_invalidate$$;
```

Comment:
We shouldn't change old scripts, since they will not change for users who are already running Citus who will only run the recent upgrade scripts. I guess we'll need to redefine these functions as well, but can drop the old versions.